composer.json

```diff
@@ -18,6 +18,6 @@
         }
     },
     "scripts": {
-        "post-create-project-cmd": "ofc\\BetterWordpress::setup"
+        "post-create-project-cmd": "ofc\\RadThemeEngine::setup"
```

Comment:
I think you'll need to make sure that the setup class on the RAD repo is named correctly since you made this change